scripts/pyenv_setup.sh

```diff
@@ -108,13 +108,10 @@ setup_pyenv() {
 
   # If the script is called with the "dot space right" approach (. ./scripts/pyenv_setup.sh),
   # the effects of this will be persistent outside of this script
-  echo "Activating pyenv"
+  echo "Activating pyenv and validating Python version"
   eval "$(pyenv init -)"
   python_version=$(python -V | sed s/Python\ //g)
-  [[ $python_version != $(cat .python-version) ]] && echo "Wrong Python version: $python_version" && exit 1
-  # The Python version installed via pyenv does not come with wheel pre-installed
-  # Installing wheel will speed up installation of Python dependencies
-  PIP_DISABLE_PIP_VERSION_CHECK=on pip install wheel
+  [[ $python_version == $(cat .python-version) ]] || (echo "Wrong Python version: $python_version" && exit 1)
```

Comment:
I had to change this condition since it would think that the script ends in a failure (it considers the $? of the last step as the exit code):
<img width="620" alt="image" src="https://user-images.githubusercontent.com/44410/107818566-d34ad580-6d45-11eb-9cb6-75de1b0e8624.png">

Reply:
Can we provide a message to instruct the user how to proceed if python version is mismatched?